src/controllers/todo.controller.ts

```diff
@@ -0,0 +1,64 @@
+import { post, param, get, put, patch, del } from '@loopback/rest';
+import { TodoSchema, Todo, TodoItem } from '../models';
+import { repository, EntityCrudRepository } from '@loopback/repository';
+
+export class TodoController {
+  @repository('todo') todoRepo: EntityCrudRepository<Todo, number>;
```

Comment:
Can we do this at the constructor level? See https://github.com/strongloop/loopback-next-hello-world/pull/23#discussion_r142416343